consensus/parlia/parlia.go

```diff
@@ -927,6 +933,13 @@ func (p *Parlia) Prepare(chain consensus.ChainHeaderReader, header *types.Header
 	nextForkHash := forkid.NextForkHash(p.chainConfig, p.genesisHash, number)
 	header.Extra = append(header.Extra, nextForkHash[:]...)
 
+	// update validators every day
+	if p.chainConfig.IsFusion(header.Number) && header.Time%86400 < 3 {
+		if err := p.updateValidatorSet(header.ParentHash); err != nil {
```

Comment:
It is the wrong place to call  `updateValidatorSet`.

Reply:
Done